src/index.ts

```diff
@@ -1,18 +1,32 @@
+import 'core-js/es/string/starts-with';
+import 'core-js/es/array/from';
+import 'core-js/es/typed-array/slice';
+import 'core-js/es/array/includes';
+import 'core-js/es/promise';
 import 'fast-text-encoding';
+import 'unfetch/polyfill/index';
+
 import Auth0Client from './Auth0Client';
 import * as ClientStorage from './storage';
 
 //this is necessary to export the type definitions used in this file
 import './global';
 
 export default async function createAuth0Client(options: Auth0ClientOptions) {
+  if (!window.crypto && (<any>window).msCrypto) {
+    (<any>window).crypto = (<any>window).msCrypto;
+  }
+  if (!window.crypto) {
+    throw new Error(
+      'For security reasons, `window.crypto` is required to run `auth0-spa-js`.'
```

Comment:
Isn't this breaking?

Reply:
No. This was already inside our `legacy` bundle. I just moved it here because we're now shipping everything IE11 compatible. <IE10 doesn't have window.crypto available, so it wouldn't work anyway.